docs/user-guide/creating-application/secrets.md

```diff
@@ -97,12 +97,49 @@ You can install `External Secrets Operator` using charts store:
 3. If you don't find any chart with this name i.e `external-secrets`, add chart repository using repository url  ` https://charts.external-secrets.io`. Please follow this [documentation](https://docs.devtron.ai/getting-started/global-configurations/chart-repo#add-chart-repository) for adding chart repository.
 4. Deploy the chart.
 
+#### AWS Secrets Manager
+
+To add secrets from AWS secret manager, navigate to `Secrets` of the application and follow the steps mentioned below :
+
+1. Create generic secret in the namespace using AWS access-key and secret-access-key.
```

Comment:
Add the graphical way to create a secret using the generic-helm-chart.

Reply:
done